README.md

```diff
@@ -1,697 +1 @@
-        wee()
-  }
-
-  // GOOD
-  function test () {
-    yey()
-  }
-  ```
-
-  - Spacing on object keys should be after the colon and not before[key-spacing]
-
-  ```js
-  // BAD
-  const obj = { foo :5 }
-  const a = { poke : 'er' }
-
-  // GOOD
-  const obj = { foo: 'bar' }
-  ```
-
-  - Spacing on keywords should be before and after[keyword-spacing]
-
-  ```js
-  // BAD
-  if(test) { ... }
-  function fn() { ... }
-  if (true){ ... }
-
-  // GOOD
-  if (test) { ... }
-  function fn () { ... }
-  ```
-
-  - No multiple empty lines[no-multiple-empty-lines]
-
-  ```js
-  // BAD (MAXIMUM 2)
-  const a = 5
-
-
-
-  cosnt b = 3
-
-  // OKAY
-  const a = 5
-
-
-  const b = 3
-
-  // GOOD
-  const a = 5
-
-  const b = 4
-  ```
-
-  - There should be a space after open curly brace and before close curly brace for objects and destructor[object-curly-spacing]
-
-  ```js
-  // BAD
-  const obj = {foo: 'bar'}
-  const a = {foo: ['test'] }
-  const b = ({foo}) => { ... }
-
-  // GOOD
-  const a = { foo: 'bar' }
-  const b = { foo: ['test']}
-  const c = ({ foo }) => { ... }
-  ```
-
-  - There should be a space before and after a block[space-before-blocks]
-
-  ```js
-  // BAD
-  if (a){
-    c()
-  }else {
-    t()
-  }
-
-  function a(){ .. }
-
-  // GOOD
-  if (a) {
-    d()
-  } else {
-    g()
-  }
-
-  function a() { ... }
-  ```
-
-  - There should be space before and after a function parenthesis[space-before-function-paren]
-
-  ```js
-  // BAD
-  function a() { ... }
-  class Foo {
-    constructor() {
-      // DO SOMETHING
-    }
-  }
-
-  // GOOD
-  function a () { ... }
-  class Foo {
-    constructor () { ... }
-  }
-  ```
-
-  - There should a space before and after an arrow - ES6[arrow-spacing]
-
-  ```js
-  // BAD
-  const a = ()=>{}
-
-  // GOOD
-  const a = () => {}
-  ```
-
-## Code Structure
-
-  - No unreachable code[no-unreachable]
-
-  ```js
-  // BAD
-  function fn () {
-    x = 1
-    return x
-    x = y
-  }
-  ```
-
-  - All require statements should be arranged*[no-mixed-requires]
-
-  ```js
-  // FOR THIS RULE THERE IS ALSO AN OPTION TO ARRANGE THE REQUIRE STARTING FROM CORE, MODULE, FILE, etc. Should we enable this?
-  // BAD
-  const fs = require('fs')
-  const async = require('async')
-  const file = require('./file')
-  const a = 'hello'
-
-  // GOOD
-  const fs = require('fs')
-  const async = require('async')
-  const file = require('./file')
-  
-  const a = 'hello'
-  ```
-
-  - Do not use `new` on requires*[no-new-require]
-
-  ```js
-  // BAD
-  const app = new require('./App')
-
-  // GOOD
-  const App = require('./App')
-  const app = new App()
-  ```
-
-  - Expression inside braces or bracket should have 2 spaces and inlined with the expression[brace-style]
-
-  ```js
-  // BAD
-  if (foo)
-  {
-    bar()
-  }
-  else {
-  test()
-  }
-
-  // GOOD
-  if (foo) {
-    bar()
-  } else {
-    test()
-  }
-  ```
-
-  - Comma should be on the last line*[comma-style]
-
-  ```js
-  // I THINK SADOHT WON'T LIKE THIS
```

Comment:
haha definitely. I downvote it :-1: 
I think it helps to avoid forgetting the fucking commas (my most common mistake), and also helps to add and remove element from the object array while developing. But if you guys hate it that much I can adapt.